src/hammer-vlsi/technology/nangate45/defaults.yml

```diff
@@ -0,0 +1,61 @@
+# Settings for the nangate45 technology to be overriden by the project.
+technology.nangate45:
+
+# Set some defaults for this technology.
+# For quick performance evaluations, use the typical voltage+temperature corners.
+# Nominal voltage for 7nm technology is 0.70 V and is the only voltage provided
+# for normal SRAMs.
+# TODO: add default temperature corners.
+vlsi:
+  # Technology dimension
+  core.node: 45
+  inputs:
+    # Supply voltages
+    supplies:
+      VDD: "1.1 V"
+      GND: "0 V"
+    # mmmc corners config.
+    mmmc_corners: [
+      {
+        "name": "PVT_1P10V_25C",
+        "type": "setup",
+        "voltage": "1.10 V",
+        "temp": "25 C"
+      }
+    ]
+  technology:
+    # Set standard cell LEF placement site
+    placement_site: "FreePDK45_38x28_10R_NP_162NW_34O"
+
+    # Set the layer that blocks vias under bumps (NOTE: no bumps used)
+    #bump_block_cut_layer: "V9"
+
+    # Set the interval and offset for tap cells (NOTE: no tap cells)
+    #tap_cell_interval: "50"
+    #tap_cell_offset: "10.564"
+
+
+technology.core:
+  # This key should exist in the stackups list in the tech json
+  stackup: "nangate45_3Ma_3Mb_2Mc_2Md"
```

Comment:
Is this the right set of layers for the stackup?

Reply:
i ran the dump_stackup_to_json.tcl in innovus on the tech-lef, so they should be close to correct

the stackup name is arbitrary. there is no official naming convention for openroad's nangate45